web/packages/teleport/src/Navigation/ResourcesSection.tsx

```diff
@@ -323,59 +329,70 @@ export function ResourcesSection({
-            ))}
+          <Box
+            css={`
+              overflow-y: auto;
+              padding: 3px;
+            `}
+          >
+            <RightPanelHeader
+              title={section.category}
+              stickyMode={stickyMode}
+              toggleStickyMode={toggleStickyMode}
+              canToggleStickyMode={canToggleStickyMode}
+            />
+            {subsections
+              .filter(section => !section.subCategory)
+              .map(section => (
+                <SubsectionItem
+                  $active={section.customRouteMatchFn(currentViewRoute)}
+                  to={section.route}
+                  key={section.title}
+                  onClick={section.onClick}
+                  exact={section.exact}
+                >
+                  <section.icon size={16} />
+                  <Text typography="body2">{section.title}</Text>
+                </SubsectionItem>
+              ))}
 
-          <Divider />
-          <Flex py={verticalPadding} px={3}>
-            <Text typography="h3" color="text.slightlyMuted">
-              Filtered Views
-            </Text>
-          </Flex>
+            <Divider />
+            <Flex py={verticalPadding} px={3}>
+              <Text typography="h3" color="text.slightlyMuted">
+                Filtered Views
+              </Text>
+            </Flex>
 
-          {subsections
-            .filter(
-              section =>
-                section.subCategory ===
-                CustomNavigationSubcategory.FilteredViews
-            )
-            .map(section => (
-              <SubsectionItem
-                $active={section.customRouteMatchFn(currentViewRoute)}
-                to={section.route}
-                key={section.title}
-                onClick={section.onClick}
-                exact={section.exact}
-              >
-                <section.icon size={16} />
-                <Text typography="body2">{section.title}</Text>
-              </SubsectionItem>
-            ))}
-        </Box>
+            {subsections
+              .filter(
+                section =>
+                  section.subCategory ===
+                  CustomNavigationSubcategory.FilteredViews
+              )
+              .map(section => (
+                <SubsectionItem
+                  $active={section.customRouteMatchFn(currentViewRoute)}
+                  to={section.route}
+                  key={section.title}
+                  onClick={section.onClick}
+                  exact={section.exact}
+                >
+                  <section.icon size={16} />
+                  <Text typography="body2">{section.title}</Text>
+                </SubsectionItem>
+              ))}
+          </Box>
+          {showPoweredByLogo && <PoweredByTeleportLogo />}
+          {cfg.edition === 'oss' && !showPoweredByLogo && <AGPLFooter />}
+          {cfg.edition === 'community' && !showPoweredByLogo && (
+            <CommunityFooter />
```

Comment:
Extracting these conditions into a separate component would improve readability and eliminate code duplication (`Section.tsx`).